### ISSUE TITLE
Could not load file or assembly Microsoft.Bcl.AsyncInterfaces

### ISSUE THREAD

**Issue by rkm** [HicServices/RDMP#1830]
**Describe the bug**

When launching the RDMP CLI on Linux, many warnings are printed about a missing `Microsoft.Bcl.AsyncInterfaces` assembly.

**To Reproduce**

On Ubuntu 22.04

1. Download & unzip rdmp-8.1.5-cli-linux.tar.xz
2. Run any command e.g. ` ./rdmp --dir ../rdmp-yaml --command run List`

**Expected behavior**

- Required assemblies are available as part of the tarball
- Messages are only printed once

**RDMP Version**

```console
$ ./rdmp --version
2024-05-17 14:56:53.3740 INFO Dotnet Version:8.0.3 . 
2024-05-17 14:56:53.3880 INFO RDMP Version:8.1.5.0 . 
rdmp 8.1.5
2024-05-17 14:56:53.3961 INFO Exiting with code 0 . 
```

**Error with Stack Trace**
```
$ ./rdmp --dir ../rdmp-yaml --command run List
2024-05-17 14:56:41.9032 INFO Dotnet Version:8.0.3 . 
2024-05-17 14:56:41.9183 INFO RDMP Version:8.1.5.0 . 
System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types.
Could not load file or assembly 'Microsoft.Bcl.AsyncInterfaces, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. The system cannot find the file specified.

Could not load file or assembly 'Microsoft.Bcl.AsyncInterfaces, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. The system cannot find the file specified.

Could not load file or assembly 'Microsoft.Bcl.AsyncInterfaces, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. The system cannot find the file specified.

... (repeated ~50 times)

ExternalDatabaseServer:1:New ExternalDatabaseServer 2f158675-7d13-431b-9477-0bae9b199308
ExternalDatabaseServer:1:New ExternalDatabaseServer 2f158675-7d13-431b-9477-0bae9b199308
```

Note that the same `ExternalDatabaseServer` is listed twice, so all the output seems to be duplicated for some reason.

**Database Engine**

N/A

**Additional context**

N/A

**Comment by jas88** [comment]
@rkm I _believe_ this is fixed in `develop` now so should work in the next release, 8.1.6.

**Comment by rkm** [comment]
Confirmed fixed with 8.1.6. Thanks!
